quickstart-python/README.md

```diff
@@ -0,0 +1,98 @@
+3. Install the required packages using pip:
+
+```bash
+pip install -r requirements.txt
+```
+
+## Running the Application
+
+To run the application, use the following command in the project directory:
+
+```bash
+python main.py
+```
+
+The application will start running at `http://0.0.0.0:8080`.
+
+## Features
+
+This app showcases how to use various Monday app functionalities, including:
+
+* Using Monday apps storage
+* Utilizing Monday apps Secrets / Environment variables
+* Using Monday apps Queue
+* Implementing OAuth process
+* Authenticating with JWT (JSON Web Tokens)
+* Creating custom actions and triggers
+* Implementing an integration recipe
+* Deploying the app to Monday code
+
+## Setting up the app in Monday
+
+Follow the instructions listed in the [SETUP.md](SETUP.md) file
+
+## Flow and Usage
+
+### NOTE:
+
+    This project is only a demonstration of how to use various Monday Code app functionalities.
+    It is not intended for production use without further modifications.
+    Please ensure to review and update the code as necessary to fit your specific needs and requirements
+    Remember to keep your OAuth credentials secure and do not expose them in your code or version control system. Use environment variables or other secure methods to handle sensitive data.
+
+### Authorization
+
+1. A user adds your recipe from the automation center.
+2. Monday sends a request to your app's authorization URL as configured in `Feature Details`.
+3. If the user has not yet authorized the app, the app redirects the user to Monday's OAuth authorization page.
+4. The user logs in and authorizes the app.
+5. Monday sends the user back to the app's callback URL with an authorization code.
+6. The app exchanges the authorization code for an access token, This token is then used for subsequent requests to the
+   server.
+7. The app saves the access token using Monday Code StorageAPI.
+
+### Custom Action
+
+8. The user can now use the custom action in their workflows.
+9. The app receives a request from Monday when the custom action should be executed.
+10. The apps gets the access token from Monday Code StorageAPI, using the user's context.
+11. The apps sends the requested action to Monday Code Worker Queue, to be executed asynchronously.
+12. The app sends a response to Monday, indicating that the action has been received and will be processed.
+
+### Worker Queue
+
+13. The worker queue sends a request to the app to process the action.
+14. The app processes the action and sends an email.
+
+## Services
+
+The project includes the following services:
+
+- `MondayApi`: Handles interactions with the Monday API.
+- `JWTService`: Handles JSON Web Token operations.
+- `SecretService`: Manages secrets within the application.
+- `StorageService`: Manages storage operations.
+- `QueueService`: Manages queue operations.
+- `SecureStorage`: Handles secure storage operations.
+- `MailService`: Handles mail operations.
```

Comment:
This creates a false believe that we support sending emails OOTB as part of monday code utils